app/src/main/res/layout/fragment_splash.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+             android:background="@color/colorPrimary"
```

Comment:
Nit: Please reformat this code for consistent spacing.

Reply:
Code reformatting done